index1.html

```diff
@@ -33,74 +33,85 @@
 	<p class="chromeframe">You are using an outdated browser. <a href="http://browsehappy.com/">Upgrade your browser today</a> or <a href="http://www.google.com/chromeframe/?redirect=true">install Google Chrome Frame</a> to better experience this site.</p>
```

Comment:
Why are there multiple index.html files?? index, index1, index2 etc?

Reply:
its there in the master, different types of Home page. were are using only index.html. its there. so that I might us it some of the components for future styling and reference. but it will be removed in the final version